hoot-core/src/main/cpp/hoot/core/algorithms/Sparse2dMatrix.cpp

```diff
@@ -37,9 +37,6 @@ Sparse2dMatrix::Sparse2dMatrix()
 
 QString Sparse2dMatrix::toString() const
 {
-  QString result;
-  result += "{";
-
```

Comment:
There are quite a few changes like this one where there is a variable that is declared, assigned, and then never used.  Deleted them.